### ISSUE TITLE
"vu" produces 「う゛」 instead of 「ゔ」

### ISSUE THREAD

**Issue by Kyuuhachi** [ueno/libkkc#7]
Inputting "vu" in hiragana produces 「う゛」 `HIRAGANA LETTER U` + `KATAKANA-HIRAGANA VOICED SOUND MARK`, instead of the more correct (and better-looking) 「ゔ」 `HIRAGANA LETTER VU`. Not a major problem since 「ゔ」 isn't often used in hiragana, but still a bit inconsistent.

If I understand the code correctly, this would need minor changes in `libkkc/rom-kana-utils.vala` (line 71) and several `data/rules/*/rom-kana/default.json` files.

I could probably make a pull request, but I have no idea how those work.

**Comment by ueno** [comment]
I agree with you about this change. The only reason why we don't have ゔ is that it is not included in JIS X 0208.

**Comment by Kyuuhachi** [comment]
Ah, that's a rather good reason. I guess I'll just add `"zvu": ["", "ゔ"],` to my personal config, then.
